src/components/molecules/MapRow/MapRow.tsx

```diff
@@ -24,7 +24,7 @@ interface MapRowProps {
   ) => void;
 }
 
-export const MapRow: React.FC<MapRowProps> = ({
+export const _MapRaw: React.FC<MapRowProps> = ({
```

Comment:
Since we're using the `_` to imply that it is not the main component that should be used, I think it makes sense to keep it named the same?

```suggestion
export const _MapRow: React.FC<MapRowProps> = ({
```

Reply:
`MapRow` is also technically an inaccurate name for this, based on a failed assumption when I was originally refactoring the legacy code. This component actually refers to a 'grid cell/bubble/seat/etc' on the map, not a row.